cmd/fluent-manager/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.19.5-alpine3.16 as builder
+FROM golang:1.19.5-alpine3.17 as builder
```

Comment:
Would you help to adjust the Dockerfile like below mentioned in https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/ ?

```shell
FROM --platform=$BUILDPLATFORM golang:1.17-alpine AS build
WORKDIR /src
COPY . .
ARG TARGETOS TARGETARCH
RUN GOOS=$TARGETOS GOARCH=$TARGETARCH go build -o /out/myapp .

FROM alpine
COPY --from=build /out/myapp /bin
```

Reply:
we need:
- `FROM --platform=$BUILDPLATFORM`
- `ARG TARGETOS TARGETARCH`
- `GOOS=$TARGETOS GOARCH=$TARGETARCH`

Reply:
updated